dxw/govuk-theme
# Update node packages

## Description

As a precursor to updating `govuk-frontend` to the latest version, this PR updates the other node packages.

**Trello card:** https://trello.com/c/AsQ1F5Lw/493-upgrade-gds-repos-to-govuk-frontend-v570-or-above

## Testing

From the theme root:

- Run `npm install` to ensure the specified package versions are installed.
- Run `npm run build`. Check that the `static` directory contains all the expected files. 
- Run `npm outdated`. Confirm that the only outdated dependency is `govuk-frontend`.
